packages/cli/src/zosfiles/compare/lf-ds/LocalfileDataset.handler.ts

```diff
@@ -25,4 +25,8 @@ export default class LocalfileDatasetHandler extends CompareBaseHandler {
     public async getFile2(session: AbstractSession, args: ICommandArguments, helper: CompareBaseHelper): Promise<Buffer> {
         return await Get.dataSet(session, args.dataSetName, { ...helper.file2Options, task: helper.task });
     }
+    public async getFile3(session: AbstractSession, args: ICommandArguments, helper: CompareBaseHelper): Promise<Buffer> {
```

Comment:
I know that this is entirely semantics, but here goes:

When I first saw a new function named getFile3, my first thought was "are 3 files somehow involved"?
I then realized that your were probably just extending the existing function naming convention.

If I correctly understand that we still only have 2 files involved, but you are extending this class to support a different type of file, my suggestion would be to call your new function "getUssFile2".

Geez, this is a verbose comment for such a minor suggestion.

Reply:
Thank you for pointing this out. After looking at things more critically, I decided to import a second handler, lf-uss (which already has the correct get method for its file 2), rather than extending lf-ds. I didn't realize at the time but my creation of getFile3 really doesn't make sense with the base class these handlers are extending (CompareBase.Handler.ts)